src/functional_test_runner/__tests__/integration/with_es_archiver.js

```diff
@@ -27,23 +29,23 @@ describe('single test that uses esArchiver', () => {
     log.info('starting elasticsearch');
     log.indent(2);
 
-    const es = createEsTestCluster({
-      log: msg => log.debug(msg),
```

Comment:
Wouldn't it be cool if we could just pass the tooling log into createEsTestCluster and the indentation would just work 😄

Reply:
c536ed5a92ff1d96009ac512aae6752f9f334c07